governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,205 @@
+
+#### DAI Borrow Spread Change
+
+*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmQrkxud)\
+*Reasoning*: [Forum Post](https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\
+*Authorizing Party*: [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4)
+
+Phoenix Labs has proposed increasing the DAI borrow spread; otherwise, there would be zero-risk in farming the [airdrop](https://forum.makerdao.com/t/sparkdao-spk-pre-farming-airdrop-general-subdao-farming-overview/21595). The Proxy Spell will increase the DAI borrow spread from 0% to **0.5%**.
+
+#### Flash Loan Fee Change
+
+*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmbCDKof#poll-detail)\
+*Reasoning*: [Forum Post](https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\
+*Authorizing Party*: [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4)
+
+The Flash Loan Fee was reset to 0.09% in the pool upgrade and needs to be lowered again to its previous level. The Proxy Spell will reduce the [Flash Loan Fee](https://docs.sparkprotocol.io/developers/guides/flash-loans#flash-loan-fee) from 0.09% to **0%**.
+
+### Core Unit MKR Vesting Transfers
+
+#### A) DECO-001 Core Unit
+
+*Authorization*: [Ratified MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP36#sentence-summary)
+
+The [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23#mission) Core Unit manages the Deco fixed-rate protocol. The proposed action concerns the transfer of **125 MKR** to the Core Unit's Auditor Wallet ([0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7](https://etherscan.io/address/0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7)). If this executive proposal passes, the transfer will be executed.
+
+#### B) SES-001 Core Unit
+
+*Authorization*: [Ratified MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP17#sentence-summary)
+
+The Sustainable Ecosystem Scaling (SES) Core Unit is the Responsible Facilitator for the Support, Protocol, Stability and Accessibility Scopes. The proposed action concerns the transfer of **34.94 MKR** to the Core Unit's Auditor Wallet ([0x87acdd9208f73bfc9207e1f6f0fde906bca95cc6](https://etherscan.io/address/0x87acdd9208f73bfc9207e1f6f0fde906bca95cc6)). If this executive proposal passes, the transfer will be executed.
+
+### Scuttle MCD_CAT Contract
+
+*Reasoning*: Forum Post "[Proposal to Scuttle MCD_CAT](https://forum.makerdao.com/t/proposal-to-scuttle-mcd-cat-upcoming-executive-spell-2023-09-13/21958)"\
+*Authorizing Party*: Protocol Scope Facilitators {**LINK MISSING**}
```

Comment:
Noting the missing link, must be added before copy is approved.

Reply:
@Le-Bateleur-2023 don't resolve comments that aren't resolved. you will miss things doing this